src/molecules/TrysilPlug.js

```diff
@@ -67,32 +82,35 @@ const TrysilPlug = ({
 	column,
 	attributes,
 	float,
+	displayPlayIcon,
 }) => {
 	const ArticleComponent = column ? ArticleCol : Article;
 	return (
 		<ArticleComponent float={float} {...column} {...attributes}>
 			<PlugLink href={url}>
-				{kicker && <Kicker>{kicker}</Kicker>}
+				{kicker && <Kicker order={1}>{kicker}</Kicker>}
 				{image
 				&& (
 					<LazyProgressiveImage
+						order={2}
 						alt={title}
 						ratio={ratio}
 						offset={offset}
 						src={placeholderUrl ? placeholderUrl : image}
 						preventBlur={preventBlur}
 						fadeIn={fadeIn}
 					>
+						{displayPlayIcon && <PlayIcon />}
```

Comment:
It would be nice to have a color prop for icon

Reply:
PlayIcon pass props along to SvgIcon